src/setup.js

```diff
@@ -1,55 +1,51 @@
+const ethers = require('ethers');
 const mintAndDeposit = require('../tests/actions/mintAndDeposit');
 
 module.exports = async function(contracts, nodes, accounts, wallet) {
   const alice = accounts[0].addr;
 
   let data = contracts.token.interface.functions.addMinter.encode([alice]);
-  let tx = await contracts.governance.propose(contracts.token.address, data, { gasLimit: 2000000 });
-  await tx.wait();
-  tx = await contracts.governance.finalize();
-  await tx.wait();
-  tx = await contracts.token.mint(alice, 500000000000);
-  await tx.wait();
-  tx = await contracts.token.approve(contracts.exitHandler.address, 500000000000);
-  await tx.wait();
-  tx = await contracts.token.approve(contracts.operator.address, 500000000000);
-  await tx.wait();
+  await (
+    await contracts.governance.propose(contracts.token.address, data, { gasLimit: 2000000 })
+  ).wait();
+  await (await contracts.governance.finalize()).wait();
```

Comment:
woah! that's LISPy. Are you allergic to `then` ? :)

Reply:
how about something like this (to reduce code noise)?

```
const awaitTx = async (call) => await call.then(tx => tx.wait());
...
...
awaitTx(contracts.governance.finalize());
```

Reply:
Sure, I'm fine with both ways :)

Reply:
Still looks almost the same as we still have to resolve the promise. But I got the chance to drop  a nice comment 😈